### ISSUE TITLE
Can't disable wrap

### ISSUE THREAD

**Issue by robo2bobo** [htacg/tidy-html5#858]
Hello,

I am using Tidy under Fedora, to format HTML under PHP. Package versions are:
libtidy-5.6.0-5.fc30.x86_64
php-tidy-7.3.14-1.fc30.x86_64

Based on the [documentation](http://api.html-tidy.org/tidy/quickref_5.6.0.html#wrap) I've set wrap to zero. Unfortunately, wrap is still in effect, for example:

source:
```html
<p>this is a <b>tessst</b> <u>content</u></p>
```

output:
```html
<p>
this
is
a
<b>
tessst</b>
<u>
content</u></p>
```

If I set wrap to some high number like 1000, then the output appears correct.

Thank you.

**Comment by geoffmcl** [comment]
@robo2bobo thank you for your issue... exposing a bug in 5.6.0 tidy...

But it has already been noted, and **fixed** in a version later than 5.6.0, in fact, from tidy 5.7.9 onwards...

See issues #780, PR #705, #704, #673, and maybe others...

The only work-around, for version 5.6, is to use a large number... 

Where a `uint` is 32-bits, any value up to 2147483647... so 1000 is fine... ;=))

Or upgrade, update, your php-tidy/libtidy installation... **if possible** ...

Am closing this, as a bug fixed... but please feel free to re-open, or open a new issue... thanks....

HTH...

**Comment by robo2bobo** [comment]
Sorry I missed the closed issues.

I believe its time to request updated packages from the Fedora maintainer :)
